static/app/views/dashboardsV2/widgetLibrary/data.tsx

```diff
@@ -166,3 +166,15 @@ export const DEFAULT_WIDGETS: Readonly<Array<WidgetTemplate>> = [
     ],
   },
 ];
+
+export const getTopNConvertedDefaultWidgets = () =>
+  DEFAULT_WIDGETS.map(widget => {
+    if (widget.displayType === DisplayType.TOP_N) {
+      return {
+        ...widget,
+        displayType: DisplayType.AREA,
+        limit: 5,
```

Comment:
I changed it to use the `TOP_N` constant from `sentry/utils/discover/types` for this one because I thought the name of it in this context seems a bit more descriptive (i.e. we're converting to an area chart so we're bringing over the top_n limit) even though they're the same value